source/_integrations/music_assistant.markdown

```diff
@@ -33,6 +33,89 @@ Under normal circumstances, Home Assistant automatically discovers your running
 
 The Music Assistant media player creates media player entities for all players available in MA including those imported from Home Assistant. This is needed to provide the full functionality Music Assistant has to offer. These entities will display media information, playback progress, and playback controls.
 
+### Action `media_player.play_media`
+
+Play media hosted on a Music Assistant server on a Music Assistant player. The action configuration is as described in the [media player documentation](https://www.home-assistant.io/integrations/media_player/#action-media_playerplay_media)
+
+The `media_content_id` payload can be any of the following:
+
+- The name of a track, artist or album. (for example, "Queen")
+- A track or album combined with the artist name (for example, "Queen - Innuendo")
+- A streaming provider URI (for example, `spotify://artist/12345`)
+
+#### Examples:
+
+Play Adele's album 25
+
+```yaml
+entity_id: media_player.music_assistant_player
+media_content_type: MUSIC
+media_content_id: 'Adele - 25'
+```
+
+Play all tracks from Stevie Wonder
+
+```yaml
+entity_id: media_player.music_assistant_player
+media_content_type: MUSIC
+media_content_id: 'Stevie Wonder'
+```
+
+Play the playlist The Best of Disco
+
+```yaml
+entity_id: media_player.music_assistant_player
+media_content_type: PLAYLIST
+media_content_id: 'The Best of Disco'
+```
+
+## Additional actions
+
+### Action `music_assistant.play_media`
+
+Play media on a Music Assistant player with more fine-grained control options.
+
+| Data attribute | Required | Description                                                                                                              | Example                           |
+| -------------- | -------- | ------------------------------------------------------------------------------------------------------------------------ | --------------------------------- |
```

Comment:
Nice docs!
One question: we are leaning towards replacing wide tables by lists, because they work better on mobile. Would you consider converting those tables here?

Reply:
No obvious reason why not. Do you have an example in the existing docs?

Reply:
Here maybe? https://www.home-assistant.io/integrations/plex/#music

Reply:
> No obvious reason why not. Do you have an example in the existing docs?

ofc, sorry.
<https://www.home-assistant.io/integrations/xiaomi_miio/> has a few.
This used to be a table:
![image](https://github.com/user-attachments/assets/3894bd6f-248a-4066-b8d8-eea997ef7a03)

Or this, using a slightly different style
![image](https://github.com/user-attachments/assets/404ba5e3-760e-4f62-af73-035670963346)

Reply:
> Here maybe? https://www.home-assistant.io/integrations/plex/#music

Ah, yes. They have some examples as well.

Reply:
I'll change it to the vacuum_goto style. Standby!

Reply:
Changes made